micro-rdk-installer/src/main.rs

```diff
@@ -47,7 +49,19 @@ struct WriteCredentials {
 }
 
 #[derive(Args)]
-struct WriteFlash {}
+struct WriteFlash {
+    #[arg(long = "app-config")]
+    config: String,
+    #[arg(long = "bin")]
+    binary_path: String,
+    #[arg(long = "nvs-size", default_value = "32768")]
+    size: usize,
+    // see comment for corresponding argument in WriteCredentials
+    #[arg(long = "nvs-offset-address", default_value = "36864")]
+    nvs_offset: u64,
```

Comment:
Why does this line start with `nvs_` but up on line 58, `size` does not have an `nvs_` prefix even though the long name for it does? It feels inconsistent, but perhaps it's consistent and I just don't have the right background to recognize that.